frontend/src/screens/channels/Channels.tsx

```diff
@@ -239,9 +239,8 @@ export default function Channels() {
                       onChange={(e) => setSwapInAmount(e.target.value)}
                     />
                     <p className="text-muted-foreground text-xs p-2">
-                      The amount is set to 90% of the funds held by the
-                      counterparty in the channel with the most receiving
-                      capacity.
+                      The amount is set to 90% of the maximum receiving capacity
+                      available in one of your Lightning channels.
```

Comment:
minor: so far we write lightning with lower cap if used in a sentence acc. to the Bitcoin Design convention.